src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -89,6 +88,10 @@
     public static final String ADDITIONAL_CRON_EXPRESSION = "AdditionalCronExpression";
     public static final String GCP_PATH_TO_SERVICE_ACCOUNT = "portal.googleProjectCredentials";
 
+    public static final String PUBSUB_BIRCH_TOPIC = "";
+    public static final String PUBSUB_PROJECT_ID = "hl7-dev-274911";
+    public static final String PUBSUB_BIRCH_SUBSCRIPTION = "pegah-results-sub";
```

Comment:
Using one of my own old test subscriptions to a fake topic I created at the beginning of birch for testing.